ranger/container/settings.py

```diff
@@ -199,6 +202,9 @@ def _get_default(self, name):
                 value = self.fm.confpath('scope.sh')
                 if not os.path.exists(value):
                     value = self.fm.relpath('data/scope.sh')
+        elif name == 'date_and_time_format':
```

Comment:
Let's add defaults for both `time_format` and `date_format` too.

Reply:
Should I comment out the defaults in **ranger/config/rc.conf** as well?

Reply:
Nah, those don't hurt.

Reply:
I added the default values, but they still need a way to ensure them being loaded before date_and_time_format. Is there a way to do this currently? Or should I work on some sort on setting dependency solution?

Reply:
I'm not sure the `%`s should still be escaped at this point. Did you test with some format string that has something that conflicts with a macro, `%d` or something?